### PR TITLE
Update self_hosting.py

### PR DESCRIPTION
**Pull request by funprogrammer89**
Added a space after the period in this line, 'Another option is to run your Pynecone service in a container.For this purpose, a'...

### CODE REVIEW

File: pcweb/pages/docs/hosting/self_hosting.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def self_hosting():
         subheader("Pynecone Container Service"),
         doctext(
             "Another option is to run your Pynecone service in a container.",
```
Comment (r0b2g1t) [inline pcweb/pages/docs/hosting/self_hosting.py:106]: It is better to add the space in this line after the ```... container. "```
This is the way how the parser works.

**Review by r0b2g1t — CHANGES_REQUESTED** [review]

File: pcweb/pages/docs/hosting/self_hosting.py (hunk 1 above)
Comment (funprogrammer89) [reply pcweb/pages/docs/hosting/self_hosting.py:106]: Should I create a new pull request with the correction handled the way you suggested? Thanks!

Comment (r0b2g1t) [reply pcweb/pages/docs/hosting/self_hosting.py:106]: No, you can fix it, commit and push it in the same branch. That will add it automatically to the PR here. :)

Comment (funprogrammer89) [reply pcweb/pages/docs/hosting/self_hosting.py:106]: Awesome, will do, thanks!

**Pull request closed without merging**